### PR TITLE
Portuguese (Brazilian) improvements by:

### PR DESCRIPTION
**Pull request by magul**
* Emanoel Lopes (https://crowdin.com/profile/emanoelopes)

### CODE REVIEW

File: pt/python_installation/instructions.md [hunk 1]
```diff
@@ -56,7 +56,14 @@ data-collapse=true ces-->
     Python 3.6.1
     
 
-Se você tem instalada uma outra "versão micro" do Python, por exemplo, 3.6.0, não precisa atualizá-la. Se não tiver o Python instalado ou quiser uma versão diferente, faça assim:
+Se você tem instalada uma outra "versão micro" do Python, por exemplo, 3.6.0, não precisa atualizá-la. Se você não tem Python instalado, ou se você quer uma versão diferente, primeiro verifique qual distribuição Linux você está usando o seguinte comando:
```
Comment (izaguerreiro) [inline pt/python_installation/instructions.md]: essa frase ficou um pouco confusa

> "primeiro verifique qual distribuição Linux você está usando o seguinte comando"

sugiro o seguinte:

> "primeiro verifique qual é a sua distribuição Linux, para isso utilize o seguinte comando:"

Comment (magul) [reply pt/python_installation/instructions.md]: Hi @izaguerreiro, first of all, thanks for a review. I do not speak (or write) in Portuguese. I just pull translations from Crowdin (here: https://crowdin.com/project/django-girls-tutorial/pt-BR#) and apply them to our GitHub repo. Can You apply your improvements there (as Crowding is an ultimate place, where we store translations) so I could re-fetch them from there?

**Comment by magul** [comment]
Hi again, @izaguerreiro, did You update that sentence on Crowdin? What should we do here?

**Comment by magul** [comment]
Ok, as there's no follow up from @izaguerreiro I'm going to merge that and wait for optional fix on Crowdin.

**Pull request merged**